### PR TITLE
feat: add hex and ascii representation

### PR DESCRIPTION
**Pull request by lucascebertin**
Ok, so, sounds not that good my `chunkString`... haz 4 lines and I wish only one!
Thoughts?

### CODE REVIEW

**Comment by lucascebertin** [comment]
image

Trying to help people to understand what's going on without to run the app

File: src/editor/panel/hex/HexPanel.tsx [hunk 1]
```diff
@@ -8,43 +8,18 @@ const HexContainer = styled.div`
...
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-      <HexCell>00</HexCell>
-    </div>
+    {props.lines.map(
+      (x: string[], firstLevel: number): JSX.Element => (
```
Comment (rafaeldelboni) [inline src/editor/panel/hex/HexPanel.tsx]: What are the vars x and y, maybe rename would be better?

File: src/editor/panel/text/TextPanel.tsx [hunk 2]
```diff
@@ -9,43 +9,18 @@ const TextContainer = styled.div`
...
-      <TextCell>.</TextCell>
-      <TextCell>.</TextCell>
-      <TextCell>.</TextCell>
-      <TextCell>.</TextCell>
-      <TextCell>.</TextCell>
-    </div>
+    {props.lines.map(
+      (x: string[], firstLevel: number): JSX.Element => (
+        <div key={firstLevel}>
+          {x.map(
```
Comment (rafaeldelboni) [inline src/editor/panel/text/TextPanel.tsx]: What are the vars x and y, maybe rename would be better?

File: src/editor/HexEditor.tsx [hunk 3]
```diff
@@ -20,11 +20,21 @@ const offsetSize = 8
 const bitsCountToNexLine = 16
 const arrayInitializer = 0
 const leftBytesPadding = '0'
+const splitInChunksOfTwo = /.{2}/g
 
 const getOffsetSizeFromBuffer = (data: Buffer): number =>
   Math.ceil(data.byteLength / offsetSize)
 
-const generateHexLines = (data: Buffer): string[] =>
+const chunkString = (text: string, chunkSize: number): string[] => {
```
Comment (rafaeldelboni) [inline src/editor/HexEditor.tsx]: Maybe the name chucks will conflict with the chunks for reading the physical file, can we find a better name?

**Review by rafaeldelboni — CHANGES_REQUESTED** [review]

File: src/editor/panel/hex/HexPanel.tsx (hunk 1 above)
Comment (lucascebertin) [reply src/editor/panel/hex/HexPanel.tsx]: Fixed!

File: src/editor/panel/text/TextPanel.tsx (hunk 2 above)
Comment (lucascebertin) [reply src/editor/panel/text/TextPanel.tsx]: Fixed!

File: src/editor/HexEditor.tsx (hunk 3 above)
Comment (lucascebertin) [reply src/editor/HexEditor.tsx]: Fixed!

**Review by rafaeldelboni — APPROVED** [review]

**Pull request merged**